src/events.h

```diff
@@ -108,6 +112,16 @@ class Events
 		void eventMonsterOnDropLoot(Monster* monster, Container* corpse);
 		bool eventMonsterOnSpawn(Monster* monster, const Position& position, bool startup, bool artificial);
 
+		int32_t getScriptId(EventInfoId eventInfoId) {
```

Comment:
`return eventInfoId == EventInfoId::EVENT_INFO_ONHEAR ? info.creatureOnHear : -1;`

Reply:
`info.creatureOnHear` is -1 per default, It is not necessary.